### DISCUSSION TITLE
XML Spritesheet Reading

### DISCUSSION THREAD

**Discussion by DrakerMaker** [HaxeFlixel/flixel#2324] [Q&A] [answered]
How do I read a Spritesheet by an XML Adobe Animate generated for it?

**Comment by TheHENOOB** [comment, marked as answer]
ninjamuffin99 (Creator of Friday Night Funkin') made a tutorial on how to use Adobe Animate or Adobe Flash art:
https://ninjamuffin99.newgrounds.com/news/post/1090580

But i think that's what you need (note this is inside of a class that is extending FlxSprite)
```haxe
var tex = FlxAtlasFrames.fromSparrow('assets/images/HoboMoveSet.png', 'assets/images/HoboMoveSet.xml'); 
frames = tex;

animation.addByPrefix('idle', 'HoboIdle', 24, true); // the HoboIdle or HoboGun is the name of the movie clip for the animation
animation.addByPrefix('gun', 'HoboGun', 24, true);

animation.play('gun');
```
